api/external/envoy/api/v2/lds.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
```

Comment:
Needed in Gloo https://github.com/solo-io/gloo/blob/master/projects/gloo/pkg/xds/envoy_server_v2.go#L32